.devops/azure-pipelines-test-dependent-projects.yml

```diff
@@ -76,12 +76,14 @@ jobs:
   - script: 'git clone https://github.com/Azure/autorest.typescript.git autorest.typescript --recursive --depth 1'
     workingDirectory: $(tempDirectory)
     displayName: "clone autorest.typescript"
-  - script: 'npm install $(Build.SourcesDirectory)/$(msRestAzureJsPackageName)'
+  - pwsh: ls $(Build.SourcesDirectory)
     workingDirectory: $(repoDir)
-    displayName: 'npm install @azure/ms-rest-azure-js'
-  - script: 'npm install $(Build.SourcesDirectory)/$(msRestJsPackageName)'
+  - script: npm install $(Build.SourcesDirectory)/$(msRestJsPackageName)
     workingDirectory: $(repoDir)
     displayName: 'npm install @azure/ms-rest-js'
+  - script: npm install $(Build.SourcesDirectory)/$(msRestAzureJsPackageName)
+    workingDirectory: $(repoDir)
+    displayName: 'npm install @azure/ms-rest-azure-js' 
```

Comment:
@KarishmaGhiya Are the changes here needed even after we get #384 merged?

Reply:
@ramya-rao-a The idea was to wait until #384 gets merged and then revert the last 2 commits in this PR - rebase them and push it. But since #384 did not get merged, this PR is currently in this state and waiting

Reply:
#384 is merged now. Can we clean up and rebase?

Reply:
Done